### DISCUSSION TITLE
Unauthorized access prohibited

### DISCUSSION THREAD

**Discussion by AnilkumarITT** [Security-Onion-Solutions/securityonion#5085] [Unsupported Versions] [answered]
Hi All 
Good morning, I was installing heavy node. 
when its asking password of soremote's: its displaying as unauthrozed access prohibited 
and Could not download 2.3.60.tar.gz from manager, please check your network settings and verify the file /opt/so/repo/2.3.60.tar.gz exists on the manager.

I am trying to install 2.3.60 heavy node and even manager as 2.3.60 version
please help me. what I am missing 

thanks in advance

**Comment by weslambert** [comment]
Did you actually enter the password for soremote when prompted?

**Comment by AnilkumarITT** [comment, marked as answer]
Hi Weslambert
I was wrong I was trying to install 2.3.61 version of heavy node. my bad
I did soup and it worked
